v2/carbs/util/hydrate.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+	"github.com/ipld/go-car/v2/carbs"
```

Comment:
nit: keep the first group of imports to just the standard library. you can use gofumpt if you want to automate that :)